man/systemd.resource-control.xml

```diff
@@ -762,6 +762,50 @@ BPFProgram=bind6:/sys/fs/bpf/sock-addr-hook
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SocketBindAllow=<replaceable>BIND_RULE</replaceable></varname></term>
+        <term><varname>SocketBindDeny=<replaceable>BIND_RULE</replaceable></varname></term>
+
+        <listitem>
+          <para>Allow or deny binding a socket address to a socket by matching it with the <replaceable>BIND_RULE</replaceable> and
+          applying a corresponding action if there is a match.</para>
+
+          <para><replaceable>BIND_RULE</replaceable> describes socket properties such as <replaceable>ADDRESS_FAMILY</replaceable>
+          and <replaceable>IP_PORTS</replaceable>.</para>
+
+          <para><replaceable>BIND_RULE</replaceable> := [<replaceable>ADDRESS_FAMILY</replaceable><constant>:</constant>]<replaceable>IP_PORTS</replaceable></para>
+
+          <para><replaceable>ADDRESS_FAMILY</replaceable> := { <constant>IPv4</constant> | <constant>IPv6</constant> }</para>
+
+          <para><replaceable>IP_PORTS</replaceable> := { <replaceable>IP_PORT</replaceable> | <replaceable>IP_PORT_RANGE</replaceable> |
+          <constant>any</constant> }</para>
```

Comment:
I see a lot of effort was put into this description…
But this isn't ideal: this type of grammar description is really hard to parse for a casual user.
Essentially, a reader needs to see what they can put in a config line, but this style of documentation is more suitable for someone writing a parser.
Also, having uppercase letters is one type of "markup", and `<replaceable>` is another type of markup. There is no need to have both.

Let's leave this as is though. Those docs are usable and probably better than many parts of the docs we already have.

Reply:
Documented with more examples.